### ISSUE TITLE
-srand SEED option does not work

### ISSUE THREAD

**Issue by gahtan-syarif** [Disservin/fast-chess#270]
specifying the -srand option does not change the seed, it still uses the same default seed value of 951356066. you can see this when comparing the order of openings used when specifying a new seed using -srand and using the default seed, the order is the same.

**Comment by Disservin** [comment]
you did enable random order and the format right?
`    if (order_ == OrderType::RANDOM && type != FormatType::NONE) shuffle();`

**Comment by gahtan-syarif** [comment]
> you did enable random order and the format right? ` if (order_ == OrderType::RANDOM && type != FormatType::NONE) shuffle();`

yes, i det format=epd and order=random, and it did shuffle the book, but the order of randomization is always the same no matter the value of -srand i set it too, meaning that the seed is stuck to the default seed

**Comment by gahtan-syarif** [comment]
hmm i checked the config file and in there the seed value got changed, so the problem isnt with the cli.cpp. maybe its in the mersenne rand code itself? somehow altering the value of the seed doesnt affect its output
